packages/hash/playwright/tests/page-creation.spec.ts

```diff
@@ -110,7 +110,7 @@ test("user can create page", async ({ page }) => {
 
   await blockContextMenu
     .locator('[placeholder="Load Block from URL..."]')
-    .fill("https://blockprotocol.org/blocks/@shinypb/emoji-trading-cards");
+    .fill("https://blockprotocol.org/blocks/@hash/code");
```

Comment:
I think we used a non-default block for this because the context menu initially didn't allow users to re-add an existing block, although that isn't the case now. So we can either

1. Use a non-default block
1. Use a default block, but update the [selector](https://github.com/hashintel/hash/blob/main/packages/hash/playwright/tests/page-creation.spec.ts#L122) in the next line to check for "Re-load block" instead of "Load block"